tests/bear_deploy.py

```diff
@@ -60,7 +62,7 @@ def tick(self):
             self.context.schedule_build(self, proto)
 
 
-    def extract_supply(self, agent, time, value):
+    def extract_supply(self, agent, time, value, commod):
```

Comment:
should this function return ` commodity_supply`

Reply:
I'm not sure what you mean here. This function doesn't return anything.